docs/usage.rst

```diff
@@ -145,6 +149,60 @@ subset of the files within the that directory with the optional ``config['patch_
 All assets defined in ``config['patch_files']`` will overwrite any files specified in the
 initial conditions or forcing if they have the same target location and name.
 
+DiagTable configuration
+-----------------------
+
+The ``diag_table`` specifies the diagnostics to be output by the Fortran model. See documentation
+for the string representation of the ``diag_table``
+`here <https://mom6.readthedocs.io/en/latest/api/generated/pages/Diagnostics.html>`_. The fv3config
+package defines a python representation of this object, :py:class:`~fv3config.DiagTable`, which can
+be used to explicitly represent the ``diag_table`` within an fv3config configuration dictionary.
```

Comment:
Maybe out of scope for this PR, but we should define some kind of internal class (not a dict) which explicitly types the fully-processed configuration (which we currently use a dict for). As it is, the term "configuration dictionary" is overloaded, since it can mean the output of yaml.load (which is more loosely typed) or the processed version of this.

Reply:
Yeah agreed. Typing the whole namelist in detail might be some work (and potentially prone to bugs) but it would make configuration easier.

Reply:
I do think out of scope for this PR.